docs/external-node.md

```diff
@@ -0,0 +1,565 @@
+   `antrea-agent` binary from the `bin` directory to the Windows VM.
+
+   ```bash
+   #! /bin/bash
+   make docker-windows-bin
+   ```
+
+2. Copy `antrea-agent.conf`, `antrea-agent.kubeconfig` and `antrea-agent.antrea.kubeconfig`
+   files to the VM. Please refer to the step 2 of [Installation on Linux VM](#installation-steps-on-linux-vm)
+   section for more information.
+
+   ```powershell
+   $WIN_AGENT_CONF_PATH="C:\antrea-agent\conf"
+   New-Item -ItemType Directory -Force -Path $WIN_AGENT_CONF_PATH
+   # Copy antrea-agent kubeconfig files
+   Copy-Item .\antrea-agent.kubeconfig $WIN_AGENT_CONF_PATH
+   Copy-Item .\antrea-agent.antrea.kubeconfig $WIN_AGENT_CONF_PATH
+   # Copy antrea-agent configuration file
+   Copy-Item .\antrea-agent.conf $WIN_AGENT_CONF_PATH
+   ```
+
+3. Create `antrea-agent` service using nssm. Below is a sample snippet to start
+   `antrea-agent` as a service:
+
+   ```powershell
+   $WIN_AGENT_BIN_PATH="C:\antrea-agent"
+   $WIN_AGENT_LOG_PATH="C:\antrea-agent\logs"
+   New-Item -ItemType Directory -Force -Path $WIN_AGENT_BIN_PATH
+   New-Item -ItemType Directory -Force -Path $WIN_AGENT_LOG_PATH
+   Copy-Item .\antrea-agent.exe $WIN_AGENT_BIN_PATH
+   nssm.exe install antrea-agent $WIN_AGENT_BIN_PATH\antrea-agent.exe --config $WIN_AGENT_CONF_PATH\antrea-agent.conf --log_file $WIN_AGENT_LOG_PATH\antrea-agent.log --logtostderr=false
+   nssm.exe start antrea-agent
+   ```
+
+4. Configure environment variable `NODE_NAME` if the VM's hostname is different
+   from the name defined in the `ExternalNode` resource.
+
+   ```powershell
+   [Environment]::SetEnvironmentVariable("NODE_NAME", "vm1")
+   [Environment]::SetEnvironmentVariable("NODE_NAME", "vm1", [System.EnvironmentVariableTarget]::Machine)
+   ```
+
+## VM network configuration
+
+`antrea-agent` uses the interface IPs or name to find the network interface on
+the external Node, and then attaches it to the OVS bridge. The network interface
+is attached to OVS as uplink, and a new OVS internal Port is created to take over
+the uplink interface's IP/MAC and routing configurations. On Windows, the DNS
+configurations are also moved to the OVS internal port from uplink. Before
+attaching the uplink to OVS, the network interface is renamed with a suffix
+"~", and OVS internal port is configured with the original name of the uplink.
+As a result, IP/MAC/routing entries are seen on a network interface configuring
+with the same name on the external Node.
+
+The outbound traffic sent from the external Node enters OVS from the internal
+port, and finally output from the uplink, and the inbound traffic enters OVS
+from the uplink and output to the internal port. The IP packet is processed by
+the OpenFlow pipeline, and the non-IP packet is forwarded directly.
+
+The following diagram depicts the OVS bridge and traffic forwarding on an
+external Node:
+![Traffic On ExternalNode](assets/traffic_external_node.svg)
```

Comment:
nit: I notice there is a *lot* of whitespace around this SVG diagram. Maybe you could crop some of it?

Reply:
updated.